From 7891f1dc0a35a1c790a20b73c8d4c84ed12c1077 Mon Sep 17 00:00:00 2001
From: Felix Schwarz <felix.schwarz@oss.schwarz.eu>
Date: Fri, 29 Jun 2018 09:01:09 +0200
Subject: [PATCH] prevent deprecation warning due to invalid escape sequence in
 NewTextTemplate doc test

Previously the running the test suite triggered a deprecation warning:

NewTextTemplate (genshi.template.text)
Doctest: genshi.template.text.NewTextTemplate ... <doctest genshi.template.text.NewTextTemplate[4]>:8: DeprecationWarning: invalid escape sequence \{
  ''')
ok
---
 genshi/template/text.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/genshi/template/text.py b/genshi/template/text.py
index e2342c7..15e3bbd 100644
--- a/genshi/template/text.py
+++ b/genshi/template/text.py
@@ -100,7 +100,7 @@ class NewTextTemplate(Template):
 
     >>> tmpl = NewTextTemplate('''Dear $name,
     ... 
-    ... \{# This is a comment #}
+    ... \\{# This is a comment #}
     ... We have the following items for you:
     ... {% for item in items %}\
     ...  * $item
-- 
2.17.1

